yt_dlp/extractor/bitchute.py

```diff
@@ -109,51 +113,106 @@ def _real_extract(self, url):
 
 
 class BitChuteChannelIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?bitchute\.com/channel/(?P<id>[^/?#&]+)'
-    _TEST = {
-        'url': 'https://www.bitchute.com/channel/victoriaxrave/',
-        'playlist_mincount': 185,
+    _VALID_URL = r'https?://(?:www\.)?bitchute\.com/(?P<type>channel|playlist)/(?P<id>[^/?#&]+)'
+    _TESTS = [{
+        'url': 'https://www.bitchute.com/channel/bitchute/',
         'info_dict': {
-            'id': 'victoriaxrave',
+            'id': 'bitchute',
+            'title': 'BitChute',
+            'description': 'md5:5329fb3866125afa9446835594a9b138',
         },
-    }
+        'playlist': [
+            {
+                'md5': '7e427d7ed7af5a75b5855705ec750e2b',
+                'info_dict': {
+                    'id': 'UGlrF9o9b-Q',
+                    'ext': 'mp4',
+                    'filesize': None,
+                    'title': 'This is the first video on #BitChute !',
+                    'description': 'md5:a0337e7b1fe39e32336974af8173a034',
+                    'thumbnail': r're:^https?://.*\.jpg$',
+                    'uploader': 'BitChute',
+                    'upload_date': '20170103',
+                    'duration': 16,
+                    'view_count': int,
+                },
+            }
+        ],
+        'params': {
+            'skip_download': True,
+            'check_formats': False,
+            'match_filter': match_filter_func('id=UGlrF9o9b-Q'),
+        },
```

Comment:
Why is the test defined like this?

Reply:
When using a channel or playlist url we get more fields in the metadata. So I wanted to test the same video as in BitChuteIE.